.github/workflows/regression-report.yml

```diff
@@ -81,22 +86,21 @@ jobs:
         env:
           COMMENT_BODY: ${{ github.event.comment.body }}
         run: |
-          CONFIG_BUNDLE_PATH=$(echo "$COMMENT_BODY" | \
+          if [[ "$COMMENT_BODY" == "Github, generate report for configs in PR description" ]]; then
+            ./.ci/append-to-github-output.sh "mode" "pr_description"
```

Comment:
you mix too much in one PR, you will never pass review and review will be complicated for me, so it will be slow, nobody will be happy.

please move `"mode" "pr_description"` in separate PR, as it is just backward compatibility with original workflow, not a something new for users.

Reply:
done

Reply:
I don't see new PR and I still see `if [[ "$COMMENT_BODY" =~ ^Git[Hh]ub,\ generate\ report\ for\ configs\ in\ PR\ description$ ]]; then` in this PR.

Reply:
ok, you want to keep it, so please share evidence that it works.